DiceKitTests/Dictionary+Functions_Tests.swift

```diff
@@ -68,4 +68,12 @@ class Dictionary_Functions_Tests: XCTestCase {
         expect(mapped) == expected
     }
 
+    func test_filterValues() {
+        let original = [1: 1, 2: 4, 3: 9, 4: 16]
+        let expected = [1: 1, 2: 4]
```

Comment:
It looks like you filtered less than 3, for keys. change it up a bit so it filtered some values in between keys, to prove it didn't just cut off the end of the ordered outcomes.